components/Cta/Cta.styles.ts

```diff
@@ -0,0 +1,70 @@
+export const cta = 'su-group su-font-regular hocus:su-underline su-transition';
+
+export const ctaVariants = {
+  primary: 'su-no-underline su-leading-display su-bg-digital-red hover:su-bg-black focus-visible:su-bg-su-bg-black active:su-bg-su-bg-black su-text-white hocus:su-text-white hocus:su-border-lagunita-light su-shadow hocus:su-shadow-md hocus:su-decoration-1 focus:su-outline-none disabled:su-border-black-20 disabled:su-bg-black-20 disabled:su-text-black',
+  secondary: 'su-no-underline su-leading-display su-bg-white hocus:su-bg-white su-text-lagunita hocus:su-text-lagunita-dark su-border hocus:su-border su-rounded su-border-lagunita-light hocus:su-border-lagunita-light active:su-border-lagunita-light su-shadow hover:su-shadow-md focus:su-ring-2 active:su-ring-2 focus:su-ring-lagunita-light active:su-ring-lagunita-light hocus:su-decoration-1 focus:su-outline-none disabled:su-border-black-30 disabled:su-bg-black-20 disabled:su-text-black-80',
+  chip: 'su-rounded-full su-shadow hover:su-shadow-md focus:su-shadow active:su-shadow su-no-underline su-underline-offset-2 su-leading-display su-bg-white hocus:su-bg-lagunita-20 su-text-lagunita hocus:su-text-lagunita-dark su-border-2 su-border-lagunita-40 focus:su-ring-2 active:su-ring-2 focus:su-ring-lagunita-40 active:su-ring-lagunita-40 focus:su-outline-none hocus:su-decoration-lagunita hocus:su-decoration-1',
+  link: 'su-underline su-leading-display su-text-lagunita hocus:su-text-black su-decoration-1 hocus:su-decoration-2 focus-visible:su-ring-2 focus-visible:su-ring-lagunita-light focus-visible:su-outline-none focus-visible:su-rounded disabled:su-text-black-60 disabled:!su-bg-transparent disabled:su-no-underline',
+  action: 'su-font-semibold su-text-digital-blue-light hocus:su-text-white su-no-underline hocus:su-underline',
+  back: 'su-no-underline su-leading-none group-hocus:su-underline su-text-black hocus:su-text-digital-red-dark focus-visible:su-ring-2 focus-visible:su-ring-digital-red-light focus-visible:su-ring-offset-4 focus:su-outline-none su-rounded-[1px]',
+  'back-external': 'su-font-semibold su-no-underline su-leading-none group-hocus:su-underline hocus:su-text-black focus-visible:su-ring-2 focus-visible:su-ring-lagunita-light focus-visible:su-ring-offset-4 focus:su-outline-none su-rounded-[1px]',
+  ghost: 'su-bg-transparent su-leading-display hocus:su-bg-transparent su-text-white hocus:su-text-white su-border-2 su-border-white hocus:su-decoration-1 focus:su-outline-none',
```

Comment:
We can update some of these CTA variants as we add them into the Examples section and remove the ones we don't need